ansible/roles/install-microshift/defaults/main.yml

```diff
@@ -37,7 +37,7 @@ firewall_trusted_cidr:
   - 10.42.0.0/16
   - 169.254.169.1/32
 
-root_bin_dir: /root/bin
+root_bin_dir: /usr/bin
```

Comment:
For microshift directory, can we use $HOME or ~/microshift - not a hardcoded value?

Reply:
So this is for the `bin` dir. On AWS RHEL images $HOME/bin/ is in `$PATH` but on GCP images it's not.
That's why I changed it to something that's in the path for both.

Reply:
I meant the microshift_dir variable. Sorry, comment is in the wrong place.

Reply:
Changed `microshift_dir` to be based on `$HOME`